.github/workflows/build-and-test.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     strategy:
       matrix:
         framework:
-          - vllm
+          - dynamo
```

Comment:
I am not in love with this unless we see a future in which there are going to be building many of these containers in parallel. That is typically what you use the `strategy.matrix` for.